package.json

```diff
@@ -67,7 +67,7 @@
     "spdy": "^3.4.1",
     "strip-ansi": "^3.0.0",
     "supports-color": "^5.1.0",
-    "webpack-dev-middleware": "3.1.2",
+    "webpack-dev-middleware": "^3.1.3",
```

Comment:
Thanks for PR, please don't change `semver`, it is can break build in future

Reply:
yikes, didn't even realize I did that! sorry, corrected.